File: backend/src/main/java/com/bluebee/smartsupply/model/OrderDelivery.java
package com.bluebee.smartsupply.model;

import java.io.Serializable;
import java.sql.*;

public class OrderDelivery implements Serializable {

    private Integer orderdeliveryid;
    private Integer orderid;
    private Integer deliverytype;
    private Integer vechilecd;
    private Integer driveruserid;
    private Integer fromaddr;
    private Integer deliveryaddr;
    private Timestamp pickupts;
    private Integer status;
     
     public Integer getOrderdeliveryid(){ 
    return this.orderdeliveryid;
    }
    public void setOrderdeliveryid(Integer orderdeliveryid){ 
    this.orderdeliveryid=orderdeliveryid;
    }
    public Integer getOrderid(){ 
    return this.orderid;
    }
    public void setOrderid(Integer orderid){ 
    this.orderid=orderid;
    }
    public Integer getDeliverytype(){ 
    return this.deliverytype;
    }
    public void setDeliverytype(Integer deliverytype){ 
    this.deliverytype=deliverytype;
    }
    public Integer getVechilecd(){ 
    return this.vechilecd;
    }
    public void setVechilecd(Integer vechilecd){ 
    this.vechilecd=vechilecd;
    }
    public Integer getDriveruserid(){ 
    return this.driveruserid;
    }
    public void setDriveruserid(Integer driveruserid){ 
    this.driveruserid=driveruserid;
    }
    public Integer getFromaddr(){ 
    return this.fromaddr;
    }
    public void setFromaddr(Integer fromaddr){ 
    this.fromaddr=fromaddr;
    }
    public Integer getDeliveryaddr(){ 
    return this.deliveryaddr;
    }
    public void setDeliveryaddr(Integer deliveryaddr){ 
    this.deliveryaddr=deliveryaddr;
    }
    public Timestamp getPickupts(){ 
    return this.pickupts;
    }
    public void setPickupts(Timestamp pickupts){ 
    this.pickupts=pickupts;
    }
    public Integer getStatus(){ 
    return this.status;
    }
    public void setStatus(Integer status){ 
    this.status=status;
    }


}

File: frontend/app/src/main/java/com/asuravan/smartsupply/pojo/OrderDelivery.kt
package com.asuravan.smartsupply.pojo

import android.os.Parcel
import android.os.Parcelable
import java.sql.Timestamp

class OrderDelivery ():Parcelable{

    var orderdeliveryid: Int? = null
    var orderid: Int? = null
    var deliverytype: Int? = null
    var vechilecd: Int? = null
    var driveruserid: Int? = null
    var fromaddr: Int? = null
    var deliveryaddr: Int? = null
    var pickupts: Timestamp? = null
    var status: Int? = null

    constructor(parcel: Parcel) : this() {
        orderdeliveryid = parcel.readValue(Int::class.java.classLoader) as? Int
        orderid = parcel.readValue(Int::class.java.classLoader) as? Int
        deliverytype = parcel.readValue(Int::class.java.classLoader) as? Int
        vechilecd = parcel.readValue(Int::class.java.classLoader) as? Int
        driveruserid = parcel.readValue(Int::class.java.classLoader) as? Int
        fromaddr = parcel.readValue(Int::class.java.classLoader) as? Int
        deliveryaddr = parcel.readValue(Int::class.java.classLoader) as? Int
        status = parcel.readValue(Int::class.java.classLoader) as? Int
    }

    override fun writeToParcel(parcel: Parcel, flags: Int) {
        parcel.writeValue(orderdeliveryid)
        parcel.writeValue(orderid)
        parcel.writeValue(deliverytype)
        parcel.writeValue(vechilecd)
        parcel.writeValue(driveruserid)
        parcel.writeValue(fromaddr)
        parcel.writeValue(deliveryaddr)
        parcel.writeValue(status)
    }

    override fun describeContents(): Int {
        return 0
    }

    companion object CREATOR : Parcelable.Creator<OrderDelivery> {
        override fun createFromParcel(parcel: Parcel): OrderDelivery {
            return OrderDelivery(parcel)
        }

        override fun newArray(size: Int): Array<OrderDelivery?> {
            return arrayOfNulls(size)
        }
    }
}